### ISSUE TITLE
Speed

### ISSUE THREAD

**Issue by maylad31** [cftang0827/pedestrian_recognition#5]
Can I reduce the image size for more speed? Nice code thanks

**Comment by cftang0827** [comment]
Nope, you can’t. Since the neural network needs the fix input size. If you want to use smaller size, you’d better to retrain the model from scratch.

Thank you for your questions.

**Comment by SehgalDivij** [comment]
@maylad31 I am preparing the data for my usecase to try this out. But that could take a while.

Could you tell me how long this took to test out on your spec? And what spec you were using?

Thanks!!

**Comment by TiFu** [comment]
I just ran the code including rendering the output image with bounding boxes and person identity on an Intel(R) Core(TM) i5-4210M CPU @ 2.60GHz  at about 5 FPS (0.185s frame time).

**Comment by maylad31** [comment]
@SehgalDivij  hi, sorry for the late revert I ran it on 1060 I get around 0.07 per frame

**Comment by SehgalDivij** [comment]
Thankyou very much. That checks out. I could run it on 1050Ti at a little less. Roughly
Although, I am curious if anyone found resizing the person's crop to a standard size generates better embeddings?

**Comment by SehgalDivij** [comment]
If yes, what was the size?
